mmv1/third_party/terraform/services/container/resource_container_cluster_test.go.tmpl

```diff
@@ -3635,6 +3643,9 @@ func TestAccContainerCluster_withAutopilotResourceManagerTags(t *testing.T) {
 			{
 				Config: testAccContainerCluster_withAutopilotResourceManagerTagsUpdate1(pid, clusterName, clusterNetName, clusterSubnetName, randomSuffix),
 				Check: resource.ComposeTestCheckFunc(
+					// Small sleep first, to avoid condition where cluster is ready but underlying GCE
+					// resources apparently aren't.
+					acctest.SleepInSecondsForTest(30),
```

Comment:
Other possibility is that I ran into some sort of shorter term problem while I was testing this and that it's not actually necessary. But this did reliably seem to get rid of the issue I mentioned in the PR comments.

See notes and linked issue about whether this is really needed in the case of VCR, though.